servers/tests/org.fusesource.ide.server.tests/src/main/java/org/fusesource/ide/server/tests/util/KarafMockRuntimeCreationUtil.java

```diff
@@ -114,13 +154,17 @@ private static void createBaseKarafFolderSkeleton(IPath path) {
 		folder = path.append("etc");
 		folder.toFile().mkdirs();
 		
-		folder = path.append("lib");
+		folder = path.append("lib").append("boot");
```

Comment:
Could u explain what you mean?

Reply:
![image](https://user-images.githubusercontent.com/1105127/30742005-906d51f4-9f97-11e7-8180-17fe287e7d3a.png)

the comments above is mentioning folder structure, it doesn't contain the boot folder

Reply:
yeah, but the lib/boot folder is just a part of 4.x. It might be better to remove that comment at all given that 5.x can completely change it once again.

Reply:
I removed that comment